pyre/check.py

```
@@ -1409,6 +1409,78 @@ def fmt_time(t):
+    """The first header line carrying `prefix`, as `(value, line)`, else None.
+
+    First one wins: a second copy of the same directive is not an error for
+    the readers that use this, it is simply not read.
+    """
+    for line in _header_lines(path):
+        if line.startswith(prefix):
+            return line[len(prefix):].strip(), line
+    return None
+
+
+def _positive_float(raw, line, path, kind):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_positive_float`

(ANN202)